src/main/java/uk/gov/hmcts/reform/civil/model/dq/Respondent1DQ.java

```diff
@@ -20,13 +20,19 @@
     private final Witnesses respondent1DQWitnesses;
     private final Hearing respondent1DQHearing;
     private final SmallClaimHearing respondent1DQHearingSmallClaim;
+    private final HearingLRspec respondent1DQHearingFastClaim;
     private final Document respondent1DQDraftDirections;
     private final RequestedCourt respondent1DQRequestedCourt;
     private final HearingSupport respondent1DQHearingSupport;
     private final FurtherInformation respondent1DQFurtherInformation;
     private final WelshLanguageRequirements respondent1DQLanguage;
     private final StatementOfTruth respondent1DQStatementOfTruth;
 
+    @JsonProperty("respondent1DQHearingFastClaim")
```

Comment:
Do we need this. The @Data annotation should give a Get by default

Reply:
I am not sure about this, I just followed the practice, how it was used for other properties.

Reply:
try that out

Reply:
removed and its working fine